crates/configuration/src/lib.rs

```diff
@@ -21,7 +22,14 @@ pub trait GlobalConfig {
 
 /// Defined in scarb manifest
 /// Configuration associated with a specific package
-pub trait PackageConfig {}
+pub trait PackageConfig {
+    #[must_use]
+    fn tool_name() -> String;
```

Comment:
I don't see a case where `&str` would not be enough

Reply:
We probably would have to make it static though, no?

Reply:
Yes